Mathlib/RingTheory/WittVector/Basic.lean

```diff
@@ -158,7 +158,6 @@ private def ghostFun : 𝕎 R → ℕ → R := fun x n => aeval x.coeff (W_ ℤ
 section Tactic
 open Lean Elab Tactic
 
--- porting note: removed mathport output related to meta code.
```

Comment:
The mathport output was simply saying that mathport could not translate the tactic, so this is superfluous:

```
/- ./././Mathport/Syntax/Translate/Tactic/Mathlib/Core.lean:38:34: unsupported: setup_tactic_parser -/

/- ./././Mathport/Syntax/Translate/Expr.lean:330:4: warning: unsupported (TODO): `[tacs] -/
/- ./././Mathport/Syntax/Translate/Expr.lean:330:4: warning: unsupported (TODO): `[tacs] -/
/- ./././Mathport/Syntax/Translate/Expr.lean:330:4: warning: unsupported (TODO): `[tacs] -/
```

See [here](https://github.com/leanprover-community/mathlib4/pull/4688/files/04b3759e3d8c3a63e42eae8d87cee015f6a2410c..550dc19a16a7e458722ee1f70a9af63ffdabf0a8#diff-848ab07015c9501300900be871ebed810000f8470fb5ddbda9c161d4e3641dbaL171-L173).